src/simd_128_u32.h

```diff
@@ -190,8 +190,9 @@ inline void mul_coef_to_buf(
 
     m128i* _src = reinterpret_cast<m128i*>(src);
     m128i* _dest = reinterpret_cast<m128i*>(dest);
-    size_t _len = len / 4;
-    size_t _last_len = len - _len * 4;
+    unsigned ratio = sizeof(*_src) / sizeof(*src);
```

Comment:
`ratio`, `_len` and `_last_len` can be const, no?

Same for the other occurences.